code/javascript/assignment#1/number.js

```diff
@@ -0,0 +1,39 @@
+const one = 1
+const two = 2
+
+console.log(1 + 2);
+console.log(one + two)
+console.log(1 -2)
+
+// for the number assignment below
+
+const november = 11 
+const september = 22
+
+console.log (11*2)
+console.log(september + november)
+console.log((november * 2 + 8)/ 2 - november)
+
+// or we can use another method
+
+let birthDay = (november * 2 + 8) / 2 - november
+console.log (birthDay)
+
+//challenge 2 ********
+
+const radius = 11
+console.log(radius**2)
+
+
+// therefore this
+
+const radiusSquare = radius**2
+const area = radiusSquare * Math.PI
+
+//print
+console.log(area.toFixed(4))
```

Comment:
This needs to be reviewed because you are expected to work with this tip: Tips:

    A radius of 2 should give you 12.5664, and a radius of 3 should give you 28.2743
    Don't forget about order of operations!
    When you are ready to submit, make a pull request following the steps in our Git Steps document .  The code appears mostly correct, but the order of operations and structure could be improved for clarity and best practices . try this code

    const radius = 3; // You can change the radius to test different values
const radiusSquare = Math.pow(radius, 2); // Use Math.pow for better readability
const area = radiusSquare * Math.PI;

console.log(`Radius squared: ${radiusSquare}`);
console.log(`PI value: ${Math.PI}`);
console.log(`Area: ${area.toFixed(4)}`);

Reply:
@Dugo11  this is a tip i provided 

 ```
const radiusSquare = Math.pow(radius, 2); // Use Math.pow for better readability
const area = radiusSquare * Math.PI;

console.log(Radius squared: ${radiusSquare});
console.log(PI value: ${Math.PI});
console.log(Area: ${area.toFixed(4)});
```